src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -214,7 +195,7 @@ private boolean build(EntityPlayer player, ItemStack stack) {
                 pushUndoList(heldItem, undoState);
             }
         }
-        BuildingModes.sortByDistance(coords, player);
+        com.direwolf20.buildinggadgets.common.tools.BuildingModes.sortByDistance(coords, player);
```

Comment:
Please don't ever opt to call methods like this.

Reply:
(except when submitting a PR with patch changes to Forge ;)) XD

Reply:
Of course but as a Java standard, don't do this unless you have, and I mean Have, to. It's completely redundant

Reply:
Duplicate